docker/debian/Dockerfile

```diff
@@ -28,7 +28,9 @@ ENV MINIFI_BASE_DIR /opt/minifi
 ENV MINIFI_HOME $MINIFI_BASE_DIR/nifi-minifi-cpp-$MINIFI_VERSION
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
 
-RUN apt-get update && apt-get install -y openjdk-11-jdk openjdk-11-source libpython3.7-dev sudo git maven libarchive13 cmake
+RUN echo "deb http://deb.debian.org/debian buster-backports main" | tee -a /etc/apt/sources.list \
+    && apt-get update && apt-get install -y openjdk-11-jdk openjdk-11-source libpython3.7-dev sudo git maven libarchive13 libtool \ 
```

Comment:
Why are we installing openjdk-11 while setting JAVA_HOME to openjdk-8 three lines above?

Reply:
Not sure. I just modified the cmake source. It was changed in #1093 , and we are still using and installing java 8 in ubuntu for example.
@fgerlits what should be here?

Reply:
Java 8 was not available in Centos 8, so I changed it to Java 11, but forgot to update the JAVA_HOME line.  I would just update that to Java 11, too.

Reply:
updated in https://github.com/apache/nifi-minifi-cpp/pull/1098/commits/ca4eaa48a97ac2da56937811589de8d3c8a02161